src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/service/cas2/ApplicationService.kt

```diff
@@ -51,8 +55,16 @@ class ApplicationService(
     return applicationRepository.findAllCas2ApplicationSummariesCreatedByUser(user.id)
   }
 
-  fun getAllSubmittedApplicationsForAssessor(): List<Cas2ApplicationSummary> {
-    return applicationRepository.findAllSubmittedCas2ApplicationSummaries()
+  fun getAllSubmittedApplicationsForAssessor(page: Int?): Pair<List<Cas2ApplicationSummary>, PaginationMetadata?> {
```

Comment:
Probs not necessary for now as you're not doing anything in the way of sorting, but we have a nice `PageCriteria` data class, which makes passing around pagination and sorting data more easy